src/highlightjs/webpack.config.ts

```diff
@@ -3,14 +3,14 @@
 import path from 'path';
 import { Configuration } from 'webpack';
 
-const config: Configuration = {
+const configIife: Configuration = {
```

Comment:
There's a new library bundling tool we can use if you're interested in replacing Webpack: [tsdown](https://tsdown.dev/).